src/System.Security.Permissions/src/System.Security.Permissions.csproj

```diff
@@ -86,33 +129,13 @@
     <Compile Include="System\Security\Policy\UrlMembershipCondition.cs" />
     <Compile Include="System\Security\Policy\Zone.cs" />
     <Compile Include="System\Security\Policy\ZoneMembershipCondition.cs" />
-    <Compile Include="System\Security\Policy\CodeConnectAccess.cs" />
-    <Compile Include="System\Security\Policy\CodeGroup.cs" />
-    <Compile Include="System\Security\Policy\Evidence.cs" />
-    <Compile Include="System\Security\Policy\EvidenceBase.cs" />
-    <Compile Include="System\Security\Policy\FileCodeGroup.cs" />
-    <Compile Include="System\Security\Policy\FirstMatchCodeGroup.cs" />
-    <Compile Include="System\Security\Policy\GacInstalled.cs" />
-    <Compile Include="System\Security\Policy\GacMembershipCondition.cs" />
-    <Compile Include="System\Security\Policy\Hash.cs" />
-    <Compile Include="System\Security\Policy\HashMembershipCondition.cs" />
-    <Compile Include="System\Security\Policy\IIdentityPermissionFactory.cs" />
-    <Compile Include="System\Security\Policy\IMembershipCondition.cs" />
-    <Compile Include="System\Security\Policy\NetCodeGroup.cs" />
-    <Compile Include="System\Security\Policy\PermissionRequestEvidence.cs" />
+    <Compile Include="System\Security\PolicyLevelType.cs" />
+    <Compile Include="System\Security\SecurityContext.cs" />
+    <Compile Include="System\Security\SecurityContextSource.cs" />
     <Compile Include="System\Security\SecurityManager.cs" />
     <Compile Include="System\Security\SecurityState.cs" />
     <Compile Include="System\Security\SecurityZone.cs" />
     <Compile Include="System\Security\XmlSyntaxException.cs" />
-    <Compile Include="System\Security\PolicyLevelType.cs" />
-    <Compile Include="System\Security\SecurityContext.cs" />
-    <Compile Include="System\Security\SecurityContextSource.cs" />
-    <Compile Include="System\Security\PermissionSet.cs" />
-    <Compile Include="System\Security\NamedPermissionSet.cs" />
-    <Compile Include="System\Security\IEvidenceFactory.cs" />
-    <Compile Include="System\Security\HostSecurityManagerOptions.cs" />
-    <Compile Include="System\Security\HostSecurityManager.cs" />
-    <Compile Include="System\Security\HostProtectionException.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == 'netfx'">
     <Reference Include="mscorlib" />
```

Comment:
```
D:\j\workspace\netfx_debug_p---88b09898\Tools\partialfacades.task.targets(70,5): error : Errors were encountered when generating facade(s). [D:\j\workspace\netfx_debug_p---88b09898\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
```

I suspect this netfx error is because you need to add some more references here. Probably at least System.Drawing but perhaps others.